### DISCUSSION TITLE
123

### DISCUSSION THREAD

**Discussion by giscus** [CODe5753/giscus#9] [Comments]
# 123

2021년 공채였던 넥토리얼 1기의 합격 후기입니다. 저는 아직도 제가 왜 붙었는지 모르겠습니다. 이 글을 통해 알 수 있는 점 넥슨 공개채용 프로세스 코딩테스트 유형 면접 방식 6개월 간 인턴 후기와 정규직 전환 스토리 서류전형 복수 지원이 가능한 프로세스 특이하게도 최대 3개의 조직에 복수 지원이 가능했습니다. "내가 갖고 있는 코인이 3개나 있는데 위축되지 말고 비슷한 스택 있으면 무조건 다 넣어보자"는 생각으로 갖고 있는 3개 코인을 모두 사용했습니다. 저는 최종적으로 아래 3개의 조직에 지원했어요. 엄청난 자기소개서 양 이렇게 최대 글자가 긴 자기소개서는 처음이었습니다. 각 문항당 최대 4천 자였거든요. 하지만 글이 너무 길면 집중력도 떨어지고 장황해질 것 같아서 1천 자 내외로 쓰기로 스스로 ..

https://imksh.com/123?category=965539

**Comment by F1ANK76** [comment]
면접은 주말에 보셨나요 ??

**Reply by CODe5753** [reply]
아뇨 평일이었습니다~

**Reply by F1ANK76** [reply]
아아 네네 ! 코테 합격 발표 이후에 몇 일정도 있다가 면접을 보러가신건가요 ??

**Reply by CODe5753** [reply]
저는 1개월정도 텀이 있었던 것 같아요
그 때 당시엔 면접관분들 업무가 마비될정도로 지원자가 많았었고 어떤 분은 거의 바로 면접보신 분도 계십니다

**Reply by F1ANK76** [reply]
아아 오 네네 감사합니다 ! 코테 시험보신 이후에 몇 일 정도 있다가 결과가 나왔었나요 ??
